src/common_types.h

```diff
@@ -12,6 +12,13 @@ using s16 = std::int16_t;
 using s32 = std::int32_t;
 using s64 = std::int64_t;
 
+// Inlining
+#ifdef _WIN32
```

Comment:
Should be _MSC_VER, some crappy toolchains like gcc on Windows will break otherwise